t/ui/13-admin.t

```diff
@@ -30,18 +30,25 @@ eval 'use Test::More::Color';
 eval 'use Test::More::Color "foreground"';
 
 use File::Path qw/make_path remove_tree/;
+use Module::Load::Conditional qw/can_load/;
 
 my $test_case = OpenQA::Test::Case->new;
 $test_case->init_data;
 
 use t::ui::PhantomTest;
 
-my $driver = t::ui::PhantomTest::call_phantom();
-if ($driver) {
+# skip if phantomjs or Selenium::Remote::WDKeys isn't available
+# we set $enterkey here to avoid using an undefined bareword later if
+# WDKeys (and hence the KEYS const) is not available.
+my $driver   = t::ui::PhantomTest::call_phantom();
+my $enterkey = '';
+if ($driver && can_load(modules => {'Selenium::Remote::WDKeys' => undef,})) {
```

Comment:
I think you don't have to do this `KEYS` thing and enable `can_load` `autoload`.
`can_load(modules => {'Selenium::Remote::WDKeys' => undef}, autoload => 1)`

Untested, but parsing the code, when using `autoload` it will call `import` function which is what `use` does.
(`use package` is roughly `require $package && $package->import`);`